src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java

```diff
@@ -48,8 +48,8 @@
 	private Date endTime;
 
 	@JsonProperty(value = "status")
-	@In(allowedValues = { "passed", "failed", "stopped", "skipped", "interrupted", "cancelled" })
-	@ApiModelProperty(allowableValues = "PASSED, FAILED, STOPPED, SKIPPED, RESETED, CANCELLED")
+	@In(allowedValues = { "passed", "failed", "stopped", "skipped", "interrupted", "cancelled", "info", "warn" })
```

Comment:
New statuses does not conform the same pattern.  We have full status name for "interrupted" and "cancelled", but truncate "info" and "warn".

Reply:
It was designed to be "INFO&WARN" statuses both on the UI and back-end sides (technical requirements in JIRA task)

Reply:
OK